gfx/common/vulkan_common.c

```diff
@@ -2012,121 +2026,124 @@ bool vulkan_create_swapchain(gfx_ctx_vulkan_data_t *vk,
          swapchain_present_mode = VK_PRESENT_MODE_FIFO_RELAXED_KHR;
          break;
       }
    }
-
-   /* If still in FIFO with no swap interval, try MAILBOX */
-   for (i = 0; i < present_mode_count; i++)
+   if (!swap_interval && swapchain_present_mode == VK_PRESENT_MODE_FIFO_KHR)
    {
-      if (     !swap_interval
-            && swapchain_present_mode == VK_PRESENT_MODE_FIFO_KHR
-            && present_modes[i] == VK_PRESENT_MODE_MAILBOX_KHR)
+      for (i = 0; i < present_mode_count; i++)
       {
-         swapchain_present_mode = VK_PRESENT_MODE_MAILBOX_KHR;
-         break;
+         if (present_modes[i] == VK_PRESENT_MODE_MAILBOX_KHR)
+         {
+            swapchain_present_mode = VK_PRESENT_MODE_MAILBOX_KHR;
+            break;
+         }
       }
    }
 
-   /* Present mode logging */
+   /* Log supported/selected present modes. */
    if (vk->swapchain == VK_NULL_HANDLE)
    {
       for (i = 0; i < present_mode_count; i++)
       {
          switch (present_modes[i])
          {
-            case VK_PRESENT_MODE_IMMEDIATE_KHR:
-               RARCH_DBG("[Vulkan] Swapchain supports present mode: IMMEDIATE.\n");
-               break;
-            case VK_PRESENT_MODE_MAILBOX_KHR:
-               RARCH_DBG("[Vulkan] Swapchain supports present mode: MAILBOX.\n");
-               break;
-            case VK_PRESENT_MODE_FIFO_KHR:
-               RARCH_DBG("[Vulkan] Swapchain supports present mode: FIFO.\n");
-               break;
-            case VK_PRESENT_MODE_FIFO_RELAXED_KHR:
-               RARCH_DBG("[Vulkan] Swapchain supports present mode: FIFO_RELAXED.\n");
-               break;
-            default:
-               break;
+            case VK_PRESENT_MODE_IMMEDIATE_KHR:      RARCH_DBG("[Vulkan] Present mode: IMMEDIATE.\n"); break;
+            case VK_PRESENT_MODE_MAILBOX_KHR:        RARCH_DBG("[Vulkan] Present mode: MAILBOX.\n"); break;
+            case VK_PRESENT_MODE_FIFO_KHR:           RARCH_DBG("[Vulkan] Present mode: FIFO.\n"); break;
+            case VK_PRESENT_MODE_FIFO_RELAXED_KHR:   RARCH_DBG("[Vulkan] Present mode: FIFO_RELAXED.\n"); break;
+            default: break;
          }
       }
    }
    else
    {
       switch (swapchain_present_mode)
       {
-         case VK_PRESENT_MODE_IMMEDIATE_KHR:
-            RARCH_DBG("[Vulkan] Creating swapchain with present mode: IMMEDIATE.\n");
-            break;
-         case VK_PRESENT_MODE_MAILBOX_KHR:
-            RARCH_DBG("[Vulkan] Creating swapchain with present mode: MAILBOX.\n");
-            break;
-         case VK_PRESENT_MODE_FIFO_KHR:
-            RARCH_DBG("[Vulkan] Creating swapchain with present mode: FIFO.\n");
-            break;
-         case VK_PRESENT_MODE_FIFO_RELAXED_KHR:
-            RARCH_DBG("[Vulkan] Creating swapchain with present mode: FIFO_RELAXED.\n");
-            break;
-         default:
-            break;
+         case VK_PRESENT_MODE_IMMEDIATE_KHR:    RARCH_DBG("[Vulkan] Creating swapchain: IMMEDIATE.\n"); break;
```

Comment:
Do you really have to reformat a ton of code that has nothing to do with the fix?